### ISSUE TITLE
Google Scholar link not created in publications page

### ISSUE THREAD

**Issue by kmundnic** [academicpages/academicpages.github.io#551]
The file `_pages/publications.md` contains the following line:

```
{% if author.googlescholar %}
  You can also find my articles on <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>
{% endif %}
```
However, this is not rendered in the publications [page](https://academicpages.github.io/publications/), although `googlescholar` is set in `_config.yml` and rendered correctly in the sidebar.

**Comment by TLouf** [comment]
I encountered the same issue, turns out it's fixed by #545: what you configure in `_config.yml` is accessed by the variable `site.author`. 

So that you know why it was rendered correctly in the sidebar, it's because of [this line](https://github.com/academicpages/academicpages.github.io/blob/25c30de2b4ce3e3f23559384699bb4b9865d6473/_includes/author-profile.html#L4) in `author-profile.html`, as we enter the `else` statement because no page author has been defined.

**Comment by kmundnic** [comment]
Thank you @TLouf. I'll close the issue, since it is fixed by https://github.com/academicpages/academicpages.github.io/pull/545.
